api/src/main/java/org/openmrs/module/kenyaemr/reporting/data/converter/definition/PopulationTypeActiveDataDefinition.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.kenyaemr.reporting.data.converter.definition;
+
+import org.openmrs.module.reporting.data.BaseDataDefinition;
+import org.openmrs.module.reporting.data.person.definition.PersonDataDefinition;
+import org.openmrs.module.reporting.definition.configuration.ConfigurationPropertyCachingStrategy;
+import org.openmrs.module.reporting.evaluation.caching.Caching;
+
+/**
+ * Population Type Data Definition
+ */
+@Caching(strategy=ConfigurationPropertyCachingStrategy.class)
+public class PopulationTypeActiveDataDefinition extends BaseDataDefinition implements PersonDataDefinition {
```

Comment:
I think you can name the class properly,does PopulationTypeActive makes sense...feels like it is hanging

Reply:
Its an Active patients line list cohort. But let me call it ActivePatientsPopulationType    does that make sense?